### PR TITLE
Feature/logging

### PR DESCRIPTION
**Pull request by thomasirmer**

### CODE REVIEW

File: apeer_dev_kit/_core.py [hunk 1]
```diff
@@ -1,59 +1,71 @@
...
 
 class _core:
     def __init__(self):
+        log.basicConfig(format='%(asctime)s [ADK:%(levelname)s] %(message)s', level=log.INFO, datefmt='%Y-%m-%d %H:%M:%S')
         try:
+            log.info('Initializing')
             self._outputs = {}
             self._wfe_output_params_file = ''
             self._input_json = json.loads(os.environ['WFE_INPUT_JSON'])
+            log.info('Found environment variable WFE_INPUT_JSON to be {}'.format(self._input_json))
```
Comment (nitish-tripathi) [inline apeer_dev_kit/_core.py]: I think it is better not to write "WFE_INPUT_JSON" in log. We should abstract this information from normal user. I think better message would be something like `'Found inputs to be {}'` or something else like this

File: apeer_dev_kit/_core.py [hunk 2]
```diff
@@ -1,59 +1,71 @@
...
             return self._input_json
         except KeyError:
-            raise KeyError(
-                'Key WFE_output_params_file not found. Please add WFE_output_params_file in input json')
+            message = 'Key WFE_output_params_file not found'
+            log.error(message)
+            raise KeyError(message)
 
     def _set_output(self, key, value):
+        log.info('Set output "{}" to "{}"'.format(key, value))
```
Comment (nitish-tripathi) [inline apeer_dev_kit/_core.py]: If we adding log.info to `_set_output()` method, then shouldn't we also log in `_set_file_output()`

Comment (thomasirmer) [reply apeer_dev_kit/_core.py]: I changed `_set_file_output()` to use `_set_output()` for actually setting the output value.

Comment (nitish-tripathi) [reply apeer_dev_kit/_core.py]: ah ok I didn't realised that

**Pull request merged**